### DISCUSSION TITLE
Put it next to the current strip instead of stacking?

### DISCUSSION THREAD

**Discussion by gerroon** [tin2tin/Pallaidium#63] [Q&A]
Hi

Is it possible to put the results of the 'No input/image" at the next frame onb the timeline instead of stacking as layers?

**Comment by tin2tin** [comment]
When doing ex. text strip batches, the generated strips needs to match the time of the input strip(vertical), but the batch number generated strips will be one after another(horizontal): 

https://github.com/tin2tin/Pallaidium/assets/1322593/3aa0cbca-62e5-43f0-8fcb-761e04c2956b

**Reply by gerroon** [reply]
Thanks, I will try that. I have been using the image mode.

**Reply by tin2tin** [reply]
If you run out of channels go to the View menu and uncheck "Limit view to contents".
